masterlist.yaml

```
@@ -831,13 +831,46 @@ plugins:
       - 'Order_Of_Sanctity.ESP'
       - 'Walkers of Morrowind.esp'
 
+###### Gameplay - Quests ######
+  ### Rise of House Telvanni ###
+  - name: 'Rise of House Telvanni.esm'
+    url: [ 'https://www.nexusmods.com/morrowind/mods/27545/' ]
+    msg:
+      - <<: *alsoUseX
+        subs: [ '[BOM_ROHT_Patch.ESP](https://www.nexusmods.com/morrowind/mods/49231/)' ]
+        condition: 'file("Rise of House Telvanni.esm") and file("Beautiful cities of Morrowind.ESP")'
+
+  - name: 'BOM_ROHT_Patch.ESP'
+    url:
+      - 'https://www.nexusmods.com/morrowind/mods/27545/'
+      - 'https://www.nexusmods.com/morrowind/mods/49231/'
+    msg:
+      - <<: *useOnlyOneX
+        subs: [ 'Rise of House Telvanni' ]
+        condition: 'file("BOM_ROHT_Patch.ESP") and file("ROHT_2_0_8.esp")'
+
+  - name: 'ROHT_2_0_8.ESP'
+    url:
+      - 'https://www.nexusmods.com/morrowind/mods/48225/'
+      - 'https://www.nexusmods.com/morrowind/mods/49231/'
+    after: [ 'Beautiful cities of Morrowind.ESP' ]
+    inc: [ 'BOM_ROHT_Patch.ESP' ]
+    req:
+      - name: 'Rise of House Telvanni.esm'
+        display: '[Rise of House Telvanni.esm](https://www.nexusmods.com/morrowind/mods/27545/)'
+
 ###### Locations - Overhauls ######
   ### Beautiful cities of Morrowind ###
   - name: 'Beautiful cities of Morrowind.ESP'
     url: [ 'https://www.nexusmods.com/morrowind/mods/49231/' ]
     after:
       - 'Better Landscapes Stonewood Pass (RP Edit).esp'
       - 'Morrowind Anti-Cheese.esp'
+    msg:
+      - <<: *patchIncluded
+        subs: [ 'Rise of House Telvanni' ]
+        condition: 'active("Rise of House Telvanni.esm") and not active("BOM_ROHT_Patch.ESP")'
```

Okay, I missed having it disabled when `ROHT_2_0_8.ESP` is installed.
```suggestion
condition: 'active("Rise of House Telvanni.esm") and not active("ROHT_2_0_8.ESP") and not active("BOM_ROHT_Patch.ESP")'
```